server.js

```diff
@@ -47,6 +50,43 @@ app.get("/api/questions/", async (req, res) => {
   }
 });
 
+app.get("/api/results/:id", async (req, res) => {
+  const { id, answer } = req.params;
+  try {
+    const results = await getResultsByQuestionID(id, answer);
```

Comment:
You probably don't need "answer" here, if you're making a request based on the id
```suggestion
  const { id} = req.params;
  try {
    const results = await getResultsByQuestionID(id);
```